bin/i18n/resources/apps/external_sources/sync_out.rb

```diff
@@ -40,12 +40,10 @@ def execute
 
                 I18nScriptUtils.rename_dir(crowdin_locale_blockly_core_dir, File.join(i18n_locale_dir, BLOCKLY_CORE_DIR_NAME))
               end
-            ensure
-              mutex.synchronize do
-                I18nScriptUtils.delete_empty_crowdin_locale_dir(crowdin_locale)
 
-                progress_bar.increment
-              end
+              I18nScriptUtils.remove_empty_dir(I18nScriptUtils.locale_dir(crowdin_locale))
```

Comment:
I see here an example of `remove_empty_dir` not immediately following a `rename_dir` hence the need of two separate methods.